keras_nlp/utils/keras_utils.py

```diff
@@ -111,3 +113,60 @@ def is_xla_compatible(model):
             tf.distribute.TPUStrategy,
         ),
     )
+
+
+def print_msg(message, line_break=True):
```

Comment:
I wonder if we should export these 2 to __internal__ so our other packages can use them?

Reply:
I think we would probably still need to copy for now, as we wouldn't want the dep on tf 2.13 anytime soon.

But could be worth thinking about this yeah. Is custom summaries a need for CV or any other keras ecosystem project?